repack

```diff
@@ -109,4 +113,13 @@ foreach my $file (@{$opt{with}}) {
     rename($file, "$file.list.chroot");
 }
 
+# disable package-lists based on names passed via --without
+foreach my $file (@{$opt{without}}) {
+    my $file_with_ext = join('/', "$newdir/config/package-lists", "$file.list.chroot");
+    next if ! -e "$file_with_ext";
+
+    print STDOUT "Disabling $file_with_ext\n";
+    rename($file_with_ext, $file);
```

Comment:
$file isn't a full path here, right? Will the rename actually put the renamed file in the correct place?